README.md

```diff
@@ -59,15 +59,85 @@ To create an instance of the service:
    1. Type a unique name for the service instance in the **Service name** field. For example, type `my-service-name`. Leave the default values for the other options.
    1. Click **Create**.
 
+## Visual Recognition
+The process for authenticating with Visual Recognition has changed:
+* For new service instances, authenticate by using IAM. See [Using IAM](#using-iam). Also set the service URL by calling the set_url() method of the service instance.
```

Comment:
We need to tell them to set their URL for IAM authentication to `https://gateway.watsonplatform.net/visual-recognition/api` since the default URL is `https://gateway-a.watsonplatform.net/visual-recognition/api`

For next week's release we need to flip this since the default URL will be `https://gateway.watsonplatform.net/visual-recognition/api` and we need to tell people using the old service to use https://gateway-a.watsonplatform.net/visual-recognition/api